docs/whats-new.md

```diff
@@ -21,6 +21,9 @@ It is now possible to globally set WebGL parameters (controlling how the GPU ren
 
 Pixel sizes in `LineLayer`, `IconLayer` and `TextLayer` now match their HTML/SVG counterparts.
 
+### `is64bitEnabled()` deprecated
```

Comment:
Let's move this to Upgrade Guide. This is a very small change that only affects 64bit layer writers, if mentioned at all it should come at the end of this v6.0 section.

Reply:
OK